packages/block-library/src/navigation-link/editor.scss

```diff
@@ -1,4 +1,14 @@
 
+[data-type="core/navigation-link"] {
```

Comment:
We have that same selector further down the file, could they be merged?

Reply:
Yes - I missed it when I was looking for it. Merging it now.